docs/getting-started/using-gnokey.md

```diff
@@ -0,0 +1,777 @@
+a keypair name to use to execute the transaction:
+
+```bash
+gnokey maketx addpkg \                                                                                                                                                                                          
+-pkgpath "gno.land/p/leon/hello_world" \
+-pkgdir "." \
+-send "" \
+-gas-fee 10000000ugnot \
+-gas-wanted 200000 \
+-broadcast \
+-chainid dev \
+-remote "127.0.0.1:26657" \
+mykey
+```
+
+If the transaction was successful, you will get the following output from `gnokey`:
+
+```
+OK!
+GAS WANTED: 200000
+GAS USED:   117564
+HEIGHT:     3990
+EVENTS:     []
+```
+
+Let's analyze the output, which is standard for any `gnokey` transaction:
+- `GAS WANTED: 200000` - the original amount of gas specified for the transaction
+- `GAS USED:   117564` - the gas used to execute the transaction
+- `HEIGHT:     3990` - the block number at which the transaction was executed at
+- `EVENTS:     []` - events emitted by the transaction, in this case, none
+
+Congratulations! You have just uploaded a pure package to your local chain.
+If you wish to upload the package to a remote testnet, make sure to switch out
+the `-chainid` & `-remote` values for the ones matching your desired testnet.
+Find a list of all networks in the [Network Configuration](../reference/network-config.md)
+section.
+
+### `Call`
+
+The `Call` message type is used to call any exported function on the chain.
+You can send a `Call` transaction with `gnokey` using the following command:
+
+```bash
+gnokey maketx call
+```
+
+:::info `Call` uses gas
+
+Using `Call` to call an exported function will use up gas, even if the function
+does not modify on-chain state. If you are calling such a function, you can use
+the [`query` functionality](#query) for a read-only call which does not use gas.
+
+:::
+
+For this example, we will call the `wugnot` realm, which wraps GNOTs to a
+GRC20-compatible token called `wugnot`. We can find this realm deployed on the
+[Portal Loop](../concepts/portal-loop.md) testnet, under the `gno.land/r/demo/wugnot`.
+
+We will wrap `1000ugnot` into the equivalent in `wugnot`. To do this, we can call
+the `Deposit()` function found in the `wugnot` realm. As previously, we will
+configure the `maketx call` subcommand:
+
+```bash
+gnokey maketx call \
+-pkgpath "gno.land/r/demo/wugnot" \
+-func "Deposit" \
+-send "1000ugnot" \
+-gas-fee 10000000ugnot \
+-gas-wanted 2000000 \
+-broadcast \
+-chainid portal-loop \
+-remote "https://rpc.gno.land:443"" \
+mykey
+```
+
+In this command, we have specified three main things:
+- The path where the realm lives on-chain with the `-pkgpath` flag
+- The function that we want to call on the realm with the `-func` flag
+- The amount of `ugnot` we want to send to be wrapped, using the `-send` flag
+
+Apart from this, we have also specified the Portal Loop chain ID, `portal-loop`,
+as well as the Portal Loop remote address, `https://rpc.gno.land:443`.
```

Comment:
Please consider completely switching this tutorial to use the portal loop, and not have a random gnodev cameo

Reply:
I've removed gnodev from this and replaced it with publishing the package to the Portal LoopL: [d6fa301](https://github.com/gnolang/gno/pull/2226/commits/d6fa3010c965703db2ea085ac71e769681a95b94) & [f805189](https://github.com/gnolang/gno/pull/2226/commits/f80518980fb464e7bb0d0f3416eb6459a3d9d87b)